src/docs/assets/portal.js

```diff
@@ -8,16 +8,6 @@ const COOKIE_DOMAIN = parts.length > 1 ? `;domain=.${parts.slice(-2).join('.')}`
 const COOKIE_MAX_AGE = 33696000;
 const COOKIE_NAME = 'euconsent';
 
-const readVendorListPromise = fetch('./vendorlist.json', {
-	headers: {
-		'Accept': 'application/json',
-		'Content-Type': 'application/json'
-	}
-})
-	.then(res => res.json())
-	.catch(err => {
-		log.error(`Failed to load vendor list from vendors.json`, err);
```

Comment:
Since you removed this part of code you don't need to import log anymore - it can be removed.